### ISSUE TITLE
WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.widget.ImageButton)

### ISSUE THREAD

**Issue by petra1** [petra1/AccessibilityDemo#5]
#### Issue
`Touch inputs must have a sufficient target size.` ([TouchSizeWcag](https://accessibilityinsights.io/info-examples/android/touch-size-wcag/))
#### Target application
ch.petraaccess_for_all.accessibilitydemo
#### Element path
android.widget.ImageButton
#### How to fix
    The element has an insufficient target size (width: 47.619dp, height: 43.047dp). Set the element's minWidth and minHeight attributes to at least 44dp.
#### Environment

This accessibility issue was found using Accessibility Insights for Android 2021.518.3 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
